Hyperlint Automation: Meta Description Fixes (#20464)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/idp-integration/okta.mdx

---
pcx_content_type: how-to
title: Okta

> description: Integrate Okta with Cloudflare Zero Trust to manage user authentication using OIDC or SAML protocols. Set up SCIM for user and group synchronization.

Reviewer: ```suggestion
description: Integrate Okta as an identity provider for Cloudflare Zero Trust.
```
